### PR TITLE
Update import workflow

### PR DESCRIPTION
**Pull request by jonboiser**
Implements #1616 and #1676 

### Design gaps

1. If user is attempting to download channel id not already on device, the channel id is shown at preview stage instead of name. Waiting on availability of API to get that data from central server.
1. File size estimates for local or remote imports not available.

### Demos

import workflow

### Testing checklist
* Happy path for remote/local import
* Shows error if remote channel ID is invalid
* Shows "already installed" next to channels already on device.
* Does not show "already installed" message next to channel not on device.
* Back button on all modals do the right thing.

### CODE REVIEW

**Comment by rtibbles** [comment]
@jonboiser could you cherry-pick this commit https://github.com/learningequality/kolibri/commit/fe344bd79a75a992853ca7b6721138e4a939928a to release-v0.5.x and open a PR, don't want this to bite us in the ass down the road.

File: kolibri/plugins/management/assets/src/views/manage-content-page/index.vue [hunk 1]
```diff
@@ -124,6 +127,9 @@
             return 'wizard-import-local';
           case ContentWizardPages.EXPORT:
             return 'wizard-export';
+          case ContentWizardPages.LOCAL_IMPORT_PREVIEW:
```
Comment (christianmemije) [inline kolibri/plugins/management/assets/src/views/manage-content-page/index.vue:130]: Missing a return for this case?

File: kolibri/plugins/management/assets/src/views/manage-content-page/wizards/import-preview.vue [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+    @back="goBack"
+  >
+    <div>
+
+      <!-- LOCAL IMPORT PREVIEW -->
+      <div v-if="importSource === 'local'">
+        <p>{{ localImportPrompt }}</p>
+
+        <ul class="channel-list">
+          <li v-for="(channel, i) in channelList" :key="i" >
```
Comment (christianmemije) [inline kolibri/plugins/management/assets/src/views/manage-content-page/wizards/import-preview.vue:18]: Couldn't you use the channel id for the key instead?

File: kolibri/plugins/management/assets/src/views/manage-content-page/index.vue (hunk 1 above)
Comment (jonboiser) [reply kolibri/plugins/management/assets/src/views/manage-content-page/index.vue:130]: No, it's the same page for both cases.

File: kolibri/plugins/management/assets/src/views/manage-content-page/wizards/import-preview.vue (hunk 2 above)
Comment (jonboiser) [reply kolibri/plugins/management/assets/src/views/manage-content-page/wizards/import-preview.vue:18]: I guess so. I'm not so paranoid about the key being globally unique, though, since the list is pretty static. It's just that omitting `:key` is now flagged by eslint.

File: kolibri/plugins/management/assets/src/state/manageContentActions.js [hunk 3]
```diff
@@ -68,70 +67,81 @@ function addChannelFileSummaries(store, channelIds) {
  * @param {Object} params - data needed to execute transition
  * @returns {undefined}
  */
-function transitionWizardPage(store, transition, params) {
+export function transitionWizardPage(store, transition, params) {
```
Comment (indirectlylit) [inline kolibri/plugins/management/assets/src/state/manageContentActions.js:70]: I like these individual `export`s placed throughout the file. Reduces boilerplate compared to the single `export` object at the end.

File: kolibri/plugins/management/assets/src/views/manage-content-page/task-status.vue [hunk 4]
```diff
@@ -13,6 +13,8 @@
 
 <script>
 
+  /* eslint-env node */
```
Comment (indirectlylit) [inline kolibri/plugins/management/assets/src/views/manage-content-page/task-status.vue:16]: why is this necessary?
